### PR TITLE
feat: add zerion wallet

### PR DESCRIPTION
**Pull request by zerts**
### Description
Add support for Zerion wallet

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn file-check`, `yarn type-check` & `yarn build` to confirm there are not any associated errors
- [ ] This PR passes the Circle CI checks

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [x] send transaction
- [x] switch chains
- [x] sign message
- [x] sign typed message
- [x] disconnect

#### Tests with demo app (SDK)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

### CODE REVIEW

File: packages/injected/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.4.0",
+  "version": "2.4.0-alpha.1",
```
Comment (Adamj1232) [inline packages/injected/package.json:3]:
```suggestion
  "version": "2.5.0-alpha.1",
```

**Comment by Adamj1232** [comment]
@zerts thank you for the PR! Can you please also update the injected-wallets package within the internal demo package [here](https://github.com/blocknative/web3-onboard/blob/v2-web3-onboard-develop/packages/demo/package.json). Then with `yarn && yarn dev` confirm full usage (wallet availability check, sign, send, switch chains, disconnect) with the internal demo

**Comment by zerts** [comment]
Thank you for the review, @Adamj1232 !
I've updated repo and checked that wallet works with it

**Comment by zerts** [comment]
Hi @Adamj1232 !
I've made all checks. everything works fine.

**Review by Adamj1232 — COMMENTED** [review]
Looks good, thank you for the PR!

**Review by Adamj1232 — APPROVED** [review]
Looks good, thank you for the PR!

**Pull request merged**